config/README.md

```diff
@@ -117,7 +118,7 @@ fmt.Println(foo)
 // Output: hello
 ```
 
-To get an access to the root element use `config.Root`:
+To get an access to the root element use `Root`:
 
 ```go
 root := provider.Get(config.Root).AsString()
```

Comment:
The YAML provider stores everything as a map of values, so in this case we have a nested map from the line 109.